src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -842,6 +842,11 @@ Status MediapipeGraphExecutor::infer(const KFSRequest* request, KFSResponse* res
     }
     std::map<std::string, mediapipe::Packet> sideInputPackets{createInputSidePackets(request)};
 #if (PYTHON_DISABLE == 0)
+    if (sideInputPackets.count(INPUT_SIDE_PACKET_TAG)) {
```

Comment:
How about changing this const name? INPUT_SIDE_PACKET_TAG sounds to generic.
Maybe make it `PYTHON_SIDE_PACKET_TAG` ?